public/app/core/components/sidemenu/utils.ts

```diff
@@ -0,0 +1,8 @@
+import { getConfig } from 'app/core/config';
+
+export const getForcedLoginUrl = (url: string) => {
+  const urlObj = new URL(`${getConfig().appSubUrl}${url}`, getConfig().appUrl);
```

Comment:
why does this link include appUrl? Should only need touse appSubUrl, any reason why this link is absolute? Not sure any other link is absolute